src/frontend/src/components/user/FormPhone.tsx

```diff
@@ -0,0 +1,132 @@
+import axios from 'axios';
+import {
+   Grid,
+   Typography,
+   ListItemText,
+   Box,
+   Stack,
+   Button,
+   ListItemAvatar,
+   Avatar,
+   ListItem,
+   TextField,
+} from '@mui/material';
+import React, { useState, Dispatch, SetStateAction } from 'react';
+import PhoneIcon from '@mui/icons-material/Phone';
+
+const baseURL = 'http://127.0.0.1:5000/user';
+interface PhoneComponentProps {
+   displayPhone: string;
+   setDisplayPhone: Dispatch<SetStateAction<string>>;
+}
+
+export function FormPhone({
+   displayPhone,
+   setDisplayPhone,
+}: PhoneComponentProps) {
+   const [editPhone, setEditPhone] = useState(false);
+   const [newPhone, setNewPhone] = useState('');
+   const [success, setSuccess] = useState(true);
+
+   function changePhone(new_phone: string) {
+      const req = {
+         is_phone: true,
+         phone: new_phone,
+      };
```

Comment:
The `changePhone` and `changeEmail` are nearly identical. I feel they could be combined, or at least the post requests could be combined and put in their common parent, `User.tsx`.

Reply:
Putting the requests there was the _only_ way I could get return values from them to work given the time constraint. If you have some other ways feel free to implement it later but for now I'm leaving them there.